followers/serializers.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from rest_framework import serializers
+from authors.models import Author
+from followers.models import Follower
+from authors.serializers import AuthorSerializer
+
+class FollowerSerializer(serializers.ModelSerializer):
+    type = serializers.SerializerMethodField(read_only=True)
+    
+    class Meta:
+        model = Follower
+        fields = ['type', 'items']
+
+    def get_type(self, obj):
+        return 'followers'
+
+    def get_items(self, obj):
+        return AuthorSerializer(Author.objects.followed_set.all(), many=True).data
```

Comment:
i still dont think this return statement works, since in the models you defined related_name = "follower" or related_name="followed", line 18 should be one of the two below
```
return AuthorSerializer(Author.objects.followed.all(), many=True).data
or
return AuthorSerializer(Author.objects.follower.all(), many=True).data
```